### PR TITLE
Add new multimedia library libav version 12.3

### PR DESCRIPTION
**Pull request by fritzkink**

### CODE REVIEW

**Comment by fritzkink** [comment]
hmm obviously https://libav.org/ does not work at the moment

File: components/multimedia/libav/Makefile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+COMPONENT_VERSION=	12.3
+COMPONENT_FMRI=		multimedia/libav
+COMPONENT_SUMMARY=	A set of portable, functional and high-performance libraries for \
+	dealing with multimedia formats of all sorts
+COMPONENT_CLASSIFICATION=System/Multimedia Libraries
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	sha256:115b659022dd387f662e26fbc5bc0cc14ec18daa100003ffd34f4da0479b272e
+COMPONENT_PROJECT_URL=	https://libav.org
+COMPONENT_ARCHIVE_URL=	https://libav.org/releases/$(COMPONENT_ARCHIVE)
```
Comment (AndWac) [inline components/multimedia/libav/Makefile:29]: DNS lookup for libav.org fails:
*** Can't find libav.org: No answer

Thus, no source file can be downloaded.

**Comment by AndWac** [comment]
What about https://github.com/libav/libav ?

**Comment by fritzkink** [comment]
this fork (from ffmpeg) is dead acoording to https://en.wikipedia.org/wiki/Libav

**Pull request closed without merging**